src/modules/pivotTable/useTableClipping.js

```diff
@@ -10,16 +10,23 @@ export const useTableClipping = ({
     visualization,
 }) => {
     const scrollPosition = useScrollPosition(containerRef)
+    const showTitle = visualization.title && !visualization.hideTitle
```

Comment:
Sure, though it's used just twice I believe.

Reply:
I don't actually know when `hideTitle` is set, but it is a property that exists.  Also... there's no option for `subtitle` in the UI but it also exists in the visualization object

Reply:
(for pivot tables)

Reply:
`subtitle` is not exposed in the options UI, it's not in classic app too.
If it's supported and it's supposed to be available it's a quick fix in the options config.
@cooper-joe @janhenrikoverland was it just missing in the classic app or left out on purpose?

Reply:
Subtitle is missing in the classic PT app, I'm not sure if that was on purpose. I don't think so. The functionality of titles is unique to PT.

In theory, charts and tables should treat the auto-generation and customization of titles/subtitles the same:

- Title: default auto-generated from filter, can be overridden with custom value or removed (none).
- Subtitle: default none, can be overridden with custom value.

The old Pivot Tables app deviates from this: 
- if title is overridden to custom, subtitle is enabled and set to auto-generated filter.

So, to align this we would need to rework this functionality. I'm not sure what effect this would have on existing visualizations and users that expecting this functionality to work the way it always has with PT. It would be confusing to have two different title functionalities in a single app, so I think it would be worth aligning them.

Reply:
I factored this so it's calculated once and exposed as an engine option.  The engine supports both title and subtitle, though only title is configurable in the UI currently